### PR TITLE
Fix sentry spam errors

### LINKED ISSUE #79: CalledProcessError: Command '['/opt/sx1302/chip_id', '-d', '/dev/spidev1.2']' returned non-zero exit status 1.
Sentry Issue: [HM-PKTFWD-4](https://sentry.io/organizations/nebra/issues/2858370371/?referrer=github_integration)

```
CalledProcessError: Command '['/opt/sx1302/chip_id', '-d', '/dev/spidev1.2']' returned non-zero exit status 1.
  File "/opt/pktfwd/utils.py", line 70, in is_concentrator_sx1302
    subprocess.run(util_chip_id_cmd, capture_output=True,
  File "subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,

SX1301 detected. util_chip_id script exited with error.
```

### LINKED ISSUE #80: lora_pkt_fwd stopped with code=1.
Sentry Issue: [HM-PKTFWD-5](https://sentry.io/organizations/nebra/issues/2858370402/?referrer=github_integration)

```
lora_pkt_fwd stopped with code=1.
```

### PR DESCRIPTION
**Pull request by ilyastrodubtsev**
**Issue**

Closes #79 
Related [#139](https://github.com/NebraLtd/hm-diag/issues/139)
**How**
Add catching expected errors.

**References**
Change linter version: [link](https://github.com/github/super-linter/issues/2255)

### CODE REVIEW

File: pktfwd/utils.py [hunk 1]
```diff
@@ -73,9 +81,12 @@ def is_concentrator_sx1302(util_chip_id_filepath, spi_bus):
         return True
     # CalledProcessError raised if there is a non-zero exit code
     # https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module
+    except subprocess.CalledProcessError as e:
+        LOGGER.warning(e)
```
Comment (marvinmarnold) [inline pktfwd/utils.py:85]: Change to debug level

File: pktfwd/utils.py [hunk 2]
```diff
@@ -207,10 +218,12 @@ def retry_start_concentrator(is_sx1302, spi_bus,
         # lora_pkt_fwd exited without error. Attempt to restart the process
         # by throwing an exception, which will trigger retry.
         elif lora_pkt_fwd_proc_returncode == 0:
-            raise Exception("lora_pkt_fwd stopped without error.")
+            raise LoraPacketForwarderStopException(
```
Comment (marvinmarnold) [inline pktfwd/utils.py:221]: rename exception: LoraPacketForwarderStoppedWithoutError

File: pktfwd/utils.py [hunk 3]
```diff
@@ -207,10 +218,12 @@ def retry_start_concentrator(is_sx1302, spi_bus,
...
+                "lora_pkt_fwd stopped without error.")
 
         # lora_pkt_fwd exited with error. Restart the container by letting
         # the python application exit without error.
         else:
-            LOGGER.error("lora_pkt_fwd stopped with code=%s." %
-                         lora_pkt_fwd_proc_returncode)
+            raise LoraPacketForwarderStopWithStatusException(
```
Comment (marvinmarnold) [inline pktfwd/utils.py:227]: The old code was correct. This is a real error, we need to fix the root cause.

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: pktfwd/utils.py (hunk 1 above)
Comment (ilyastrodubtsev) [reply pktfwd/utils.py]: Level changed: https://github.com/NebraLtd/hm-pktfwd/blob/c9532394d79bdeb812eeb43a7153c7c966b41dce/pktfwd/utils.py#L81

File: pktfwd/utils.py (hunk 2 above)
Comment (ilyastrodubtsev) [reply pktfwd/utils.py]: Renamed: https://github.com/NebraLtd/hm-pktfwd/blob/c9532394d79bdeb812eeb43a7153c7c966b41dce/pktfwd/utils.py#L217

File: pktfwd/utils.py (hunk 3 above)
Comment (ilyastrodubtsev) [reply pktfwd/utils.py]: Roolback https://github.com/NebraLtd/hm-pktfwd/blob/c9532394d79bdeb812eeb43a7153c7c966b41dce/pktfwd/utils.py#L222-L224

File: pktfwd/utils.py [hunk 4]
```diff
@@ -207,7 +214,8 @@ def retry_start_concentrator(is_sx1302, spi_bus,
         # lora_pkt_fwd exited without error. Attempt to restart the process
         # by throwing an exception, which will trigger retry.
         elif lora_pkt_fwd_proc_returncode == 0:
-            raise Exception("lora_pkt_fwd stopped without error.")
+            raise LoraPacketForwarderStoppedWithoutError(
```
Comment (marvinmarnold) [inline pktfwd/utils.py:217]: Should be LoraPacketForwarderStoppedWithError

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: pktfwd/utils.py (hunk 4 above)
Comment (marvinmarnold) [reply pktfwd/utils.py:217]: Sorry this is correct.

**Review by marvinmarnold — APPROVED** [review]

**Pull request merged**